stories/components/describedselector/describedselector.stories.mdx

```diff
@@ -0,0 +1,43 @@
+import { Meta, Story, Canvas, ArgsTable } from '@storybook/addon-docs/blocks'
+import { DescribedSelector } from './DescribedSelector'
+
+<Meta title="Components/DescribedSelector" component={DescribedSelector} />
```

Comment:
Let's use a different folder one of: 
Candidates, Examples, Molecules.
Happy for there to be more than just one folder if some are better in Examples and some in Candidates or Labs etc.

Reply:
I've put them in a separate Candidates folder.